### ISSUE TITLE
Add logins support for scrubbing encrypted data

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/uniffi-rs#932]
This will be very similar to what was done in #4029 for Logins with some of the work having already been implemented in #4064. This work should be based on the logins encryption branch and merged into it once this work has been approved.

Note: This ticket is blocked by #4065.

**Comment by data-sync-user** [comment]
➤ Ben Dean-Kawamura commented:

I'm going to push this one back to next sprint.  The main encryption issue is still in my local fork, I'd rather not start this until that one gets approved and merged.
